lib/wash_out/soap.rb

```diff
@@ -30,16 +36,21 @@ def soap_action(action, options={})
             options[:to] ||= action.to_s
             action         = action.to_s.camelize
           end
+        end
 
+        if options[:request_tag].blank?
+          if options[:as].present?
```

Comment:
can you combine this if statement with the one on line 26? not a big deal, the check just feels a bit redundant